src/azure-cli/azure/cli/command_modules/rdbms/_transformers.py

```diff
@@ -56,3 +56,38 @@ def table_transform_output_list_sku(result):
                 raise CLIError("There is no sku available for this location.")
 
     return table_result
+
+
+def table_transform_output_list_servers_single_server(result):
```

Comment:
Nit : This is exactly same as table_transform_output_list_servers except haState and availablityZone. So try and see either

 1. If you can pass a parameter to the function and based on the parameter value( server/flexible server) , determine whether or not to add those two fields, or
 2. Have a common function and remove the repeated code there and make both these functions call the common function wherever applicable and add the rest of the exclusive fields in the respective caller.

Reply:
Resolved. Thanks!!